src/coordination/kubernetes/Akka.Coordination.KubernetesApi/KubernetesApi.cs

```diff
@@ -41,7 +41,7 @@ public LeaseResource(string? owner, string version, long time)
 
         public string? Owner { get; }
         public string Version { get; }
-        public long Time { get; }
+        public DateTime Time { get; }
```

Comment:
The internal lease resource timeout property now stores the absolute expiration date and time, not the milliseconds offset until lease expiration